kategory/src/main/kotlin/kategory/data/EitherT.kt

```diff
@@ -78,3 +78,51 @@ data class EitherT<F, A, B>(val MF: Monad<F>, val value: HK<F, Either<A, B>>) :
         return GA.map(fa, { EitherT(MF, MF.map(it.lower(), { it.ev() })) })
     }
 }
+
+class EitherTInstances<F, L>(val MF : Monad<F>) : EitherTMonadError<F, L> {
+    override fun MF(): Monad<F> = MF
+}
+
+interface EitherTMonad<F, L> : Monad<EitherTF<F, L>> {
```

Comment:
I'd rather we kept all instances together, yet still on a separate file. Open for discussing pros/cons.